modal.css

```diff
@@ -0,0 +1,247 @@
+.modal {
+  width: 70%;
+  height: 68%;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 0.4s ease-in-out;
+  z-index: 10;
+  border-radius: 10px;
+  background-color: #fff;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+}
+
+.modal-img-container {
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 

- Nice implementing the design, however, it would be better if you add a little bit of blur to the sub background when the popup is open in order to match the Figma design and also make a good contrast art 👨🏻‍🎨 👍🏿
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Make sure you respect you text decription `line-height and font-size`


Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/233047047-a61f99cf-b71e-4f60-9a95-7dbd872740e2.png)| ![181](https://user-images.githubusercontent.com/79058364/189106086-ad46721f-211a-41e9-97b7-421fb32231c2.png)

Reply:
- [ ] Make sure to improve this buttons section 👇🏻 By adding modre padding or margin top and make sure the `see live & see source` text are correctly align with their icons

![image](https://user-images.githubusercontent.com/79058364/233047217-f2c7814d-33e7-4c88-ae6c-101b0d2d51e7.png)

Reply:
Dear @BenMukebo ,
I made the required changes, thanks for guidance